rep-2014.rst

```diff
@@ -0,0 +1,238 @@
+
+
+Industry standards
+------------------
+There are no globally accepted industry standards for benchmarking robotic systems. The closest initiative to a standardization effort in robotics is the European H2020 Project EUROBENCH which aimed at creating the first benchmarking framework for robotic systems in Europe focusing on bipedal locomotion. The project has been completed in 2022 and the results are available in [9]_. The project has been a great success and has been used to benchmark a wide range of bidepal robotic systems throughout experiments however there're no public plans to escalate the project to other types of robots, nor the tools have been used elsewhere.
+
+
+When looking at other related areas to robotics we find the MLPerf Inference and MLCommons initiatives which are the closest to what we are trying to achieve in ROS 2. The MLPerf Inference is an open source project that aims to define a common set of benchmarks for evaluating the performance of machine learning inference engines. The MLCommons is an open source project that aims to define a common set of benchmarks for evaluating the performance of machine learning models. Both projects have been very successful and are widely used in the industry. The MLPerf Inference project has been completed in 2021 and the results inference benchmarks available in [10]_. The MLCommons project has become an industry standard in Machine Learning and the results publicly disclosed in [11]_.
+
+
+Performance metrics in robotics
+===============================
+Robots are deterministic machines and their performance should be understood by considering metrics like the following ones:
+
+- **latency**: time between the start and the completion of a task.
+
+- **bandwidth or throughput**: the total amount of work done in a given time for a task.
+
+- **power**: the electrical energy per unit of time consumed while executing a given task.
+
+- **performance-per-watt**: total amount of work (generally *bandwidth* or *throughput*) that can be delivered for every watt of power consumed.
+
+- **memory**: the amount of short-term data (not to be confused with storage) required while executing a given task.
+
+These metrics can help determine performance characteristics of a robotic system. Of most relevance for robotic systems we often encounter the *real-time* and *determinism* characteristics defined as follows: 
+
+- **real-time**: ability of completing a task's computations while meeting time deadlines
+- **determinsm**: that a task happens in exactly the same timeframe, each time.
```

Comment:
I think I understand what you mean, but "happens in exactly the same timeframe" may not be easily interpretable by people not already familiar with the concept of determinism.